testing/libosip2/APKBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: David Sugar <tychosoft@gmail.com>
+# Contributor: David Sugar <tychosoft@gmail.com>
+pkgname=libosip2
+pkgver=5.1.0
+pkgrel=0
+pkgdesc="oSIP is an implementation of SIP"
+url="https://www.gnu.org/software/osip/"
+arch="all"
+license="LGPL-2.1-or-later"
+makedepends="
+	autoconf
+	automake
+	libtool
+"
+options="libtool"
+subpackages="$pkgname-dev $pkgname-doc"
+source="
+	http://ftp.gnu.org/gnu/osip/$pkgname-$pkgver.tar.gz
+	"
```

Comment:
super nitty, but please leave a newline here.

Reply:
I took some time to look at other packages.  I used to have a patch and didn't need that anymore, so I actually should have consolidated the source line back into one line.  I also now see other packages do indeed typically use name-$pkgvar.tar.gz in source url's.  Especially to do my first packages, it did help me to compare more carefully with existing packages.

Reply:
I can recommend you to use `apkbuild-lint` from the `atools ` package which will spot these things for you :)

Reply:
Actually apkbuild-lint returned no warnings...so it did not check for that.   I originally had a bunch of cd $builddir which it did warn me about, and a few other minor things, though.